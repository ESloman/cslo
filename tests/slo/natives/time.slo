var t = time();
println(t);
