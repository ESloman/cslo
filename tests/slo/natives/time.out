1.75014e+09
